docs/apache-airflow/administration-and-deployment/logging-monitoring/check-health.rst

```diff
@@ -147,12 +147,12 @@ To check if the worker running on the local host is working correctly, run:
 
 .. code-block:: bash
 
-    celery --app airflow.executors.celery_executor.app inspect ping -d celery@${HOSTNAME}
+    celery --app airflow.providers.celery.executors.celery_executor.app inspect ping -d celery@${HOSTNAME}
```

Comment:
I am not 100% sure if I want to change the app name yet. It has some unnerving property and I am thinking that maybe we should leave it as it was (though it's a bit incosistent).

Reply:
What change did you had in mind?

Reply:
to leave "airflow.executors.celery_executor.app" app name as the name of the celery app.

Reply:
Ah. But we can't have it actually, there is a path that we have to serve here, however what I can do - I can automatically replace the old path when read and raise a deprecation warning.

Reply:
I think this one nicely gives the deprecation warning and informs what to do to remove it:

https://github.com/apache/airflow/pull/32526/files#diff-0e0d141920152b568dd569500b73ebaca58893fa2a98255ef5a565743f5e374aR71

Reply:
Also updated significant newsfragment with more detailes what should be changed (the changes should be backwards-compatible, they should just raise deprecation warnings to change the app name in configuration and Health Checks).